src/components/checkbox/checkbox.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, text, number, select
+} from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import Checkbox from './checkbox.js';
+
+storiesOf('Checkbox', module)
+  .add('default', () => {
+    const reverse = boolean('reverse', Checkbox.defaultProps.reverse);
+    const fieldHelpInline = boolean('fieldHelpInline', false);
+    const label = text('label', 'Example Checkbox');
+    const labelHelp = text('labelHelp', 'This text provides more information for the label.');
+    const inputWidth = number('inputWidth', 0, {
+      range: true,
+      min: 0,
+      max: 50,
+      step: 1
+    });
+    const fieldHelp = text('fieldHelp', 'This text provides help for the input.');
+    const labelInline = boolean('labelInline', false);
+    const labelWidth = labelInline ? number('labelWidth', 0, {
+      range: true,
+      min: 0,
+      max: 50,
+      step: 1
+    }) : undefined;
+    const labelAlign = labelInline ? select(
+      'labelAlign',
+      OptionsHelper.alignBinary,
+      OptionsHelper.alignBinary[0]
+    ) : undefined;
+
+    return (
```

Comment:
I'm concerned that this is a very large element which isn't really giving the viewer much useful information. Can we add descriptions to this?

![image](https://user-images.githubusercontent.com/26925206/53880091-06d43f80-4008-11e9-9489-1a01b636c197.png)

Reply:
@samhinton88 My PR contains the stories for Dropdown which are in a similar situation. Because the export is wrapped in a number of HOCs, Storybook doesn't seem to be able to grab the info for propTypes. I would suggest we tackle this when the component itself gets a refactor...